pactum/exporters/openapi.py

```diff
@@ -93,6 +101,19 @@ def _parameter_to_dict(self, action, parameter):
 
         return param_dict
 
+    def _querystring_to_dict(self, action, querystring):
+        qs_dict = ({
+            'name': querystring.name,
+            'in': 'query',
+            'required': querystring.required,
+            'description': querystring.__doc__
+        })
+        if issubclass(querystring.type, fields.Field):
```

Comment:
O que acontece se ela não for sub-classe?

Reply:
se não for subclasse, a chave não é adicionada ao dicionário.